cli/impl/src/main/java/io/helidon/build/cli/impl/InitCommand.java

```diff
@@ -86,7 +89,24 @@ protected void invoke(CommandContext context) throws Exception {
                 Log.info("Using Helidon version " + helidonVersion);
             } else {
                 String defaultHelidonVersion = defaultHelidonVersion();
-                helidonVersion = prompt("Helidon version", defaultHelidonVersion, this::isSupportedVersion);
+                FileSystem latestArchetype = latestArchetype();
+                ArchetypesData archetypesData = ArchetypesDataLoader.load(latestArchetype);
+                List<String> versions = archetypesData.versions();
```

Comment:
Can you add a unit test (see `InitCommandTest`).

Reply:
All tests in `InitCommandTest` related to `batch` mode. List versions is related to `interactive` mode. Is it possible to add tests for `interactive` mode?

Reply:
See `testInteractiveSe` and `testInteractiveMp` ; they use a file to simulate user input.